pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -68,6 +73,101 @@ func (alloc *backfillAction) Execute(ssn *framework.Session) {
 			}
 		}
 	}
+
+	if len(alloc.arguments) == 0 {
+		return
+	}
+
+	backfillEnabled, err := strconv.ParseBool(alloc.arguments[conf.BackfillFlagName])
+	if err != nil {
+		glog.V(3).Infof("backfilling non best effort jobs is disabled")
+		return
+	}
+
+	if backfillEnabled {
+		// Collect back fill candidates
+		backFillCandidates := make([]*api.JobInfo, 0, len(ssn.Jobs))
+		for _, job := range ssn.Jobs {
+			if !ssn.BackFillEligible(job) {
+				continue
+			}
+			backFillCandidates = append(backFillCandidates, job)
+		}
+
+		// Release resources allocated to unready top dog jobs so that
+		// we can back fill more jobs in the next step.
+		resourceReleased := false
+		for _, job := range ssn.Jobs {
+			if ssn.JobReady(job) || ssn.JobBackfillReady(job) {
+				glog.V(3).Infof("Disable backfill on job <%v/%v>", job.Namespace, job.Name)
+			} else {
+				releaseReservedResources(ssn, job)
```

Comment:
How about `resourceReleased = releaseReservedResources(ssn, job)` and add a return value for the function? According to the function, there is some case that it could not release any resource.

Reply:
I don't see the need